src/osd/OSD.cc

```diff
@@ -7155,6 +7155,13 @@ void OSD::sched_scrub()
 	pg->unlock();
 	break;
       }
+      if (pg->is_scrub_registered()) {
+	// Computing backoff based on the normal scrub interval.  By default of 24 hours
+	// we will try again in 10 minutes by dividing by 144.
+	utime_t backoff_time = utime_t(cct->_conf->osd_scrub_min_interval / 144, 0);
```

Comment:
Need to check pool min interval too and @neha-ojha suggests using 1/24 for 1 hour by default